src/status_im/subs/wallet/wallet.cljs

```diff
@@ -198,3 +198,11 @@
  (fn [[current-viewing-account network-details]]
    (let [network-preferences-names (:network-preferences-names current-viewing-account)]
      (filter #(contains? network-preferences-names (:network-name %)) network-details))))
+
+(rf/reg-sub
+ :wallet/accounts-with-customization-color
+ :<- [:wallet/accounts]
+ (fn [accounts]
+   (mapv (fn [{:keys [color] :as account}]
```

Comment:
Using lazy sequences can be beneficial if other computations will (could be in the future) performed on the result of this subscription. `mapv` will eagerly process everything, which is likely fine because `accounts` is a small data set.

Things can get confusing and inefficient if we mix them, so it's more common in Clojure that devs use the lazy ones and only use eager alternatives if really needed.